src/GraphQLMiddleware.php

```diff
@@ -67,12 +61,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
     }
 
     private function isGraphQLRequest(ServerRequestInterface $request) {
-        return $this->hasUri($request) || $this->hasGraphQLHeader($request);
```

Comment:
exactly, how this is supposed to work with a POST request with the `application/graphql` ?